src/main/java/org/veupathdb/eda/binaryfiles/printer/BinaryFilePrinter.java

```diff
@@ -32,17 +33,17 @@ public static void printBinaryFile(Path binaryFile, Path metajsonFile) {
     
     JSONObject metajson = readMetaJsonFile(metajsonFile);
     String binaryFileNm = binaryFile.toFile().getName();
-
-    if (binaryFileNm.equals(BinaryFilesManager.ANCESTORS_FILE_NAME)) 
-      printAncestorFile(binaryFile, metajson);
     
-    else {
-      BinaryConverter<?> converter = new StringValueConverter(FilesDumper.BYTES_RESERVED_FOR_ID_STRING); // assume converter for id_map file
-      if (!binaryFileNm.equals(BinaryFilesManager.IDS_MAP_FILE_NAME)) 
-        converter = getVarType(binaryFileNm, metajson).getConverterSupplier().get();     
- 
-      printVarFile(binaryFile, converter);
-    }
+    switch (binaryFileNm) {
+    case BinaryFilesManager.ANCESTORS_FILE_NAME:
```

Comment:
How are these string matches right?  There's not just _one_ ancestors file right?  Isn't it a function of entity ID?

Reply:
we're just trying to figure out what _type_ of file it is.  the metadata has the other info.

Reply:
So maybe binaryFileNm -> binaryFileType and ANCESTORS_FILE_NAME -> ANCESTORS_FILE?  Or put a comment because this is confusing.